assets/app/index.html

```diff
@@ -5,8 +5,13 @@
     <base href="/">
     <title>OpenShift Web Console</title>
     <meta name="description" content="">
-    <meta name="viewport" content="width=device-width">
-    <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" type="image/png" href="/images/favicon.png" />
```

Comment:
to test this, run in built mode and make sure everything gets requested under /console/images/...